package.json

```diff
@@ -13,7 +13,7 @@
   },
   "author": "ACME Inc.",
   "scripts": {
-    "test": "./test/spec.rb",
+    "test": "mocha test/*test.js",
```

Comment:
True this could just be `mocha`. IMHO there's a lot of value in having
opinions in boilerplates. In this specific case I don't much care.
On 17 May 2016 1:13 AM, "Hugo Giraudel" <notifications@github.com> wrote:

In package.json
<https://github.com/kaelig/sass-project-boilerplate/pull/1#discussion_r63369657>
:

> @@ -13,7 +13,7 @@
>    },
>    "author": "ACME Inc.",
>    "scripts": {
> -    "test": "./test/spec.rb",
> +    "test": "mocha test/*test.js",

Assuming the filenames will end with test is already too much for a
boilerplate in my opinion. I would simple glob everything in the test
folder. Any file that does not contain any test for Mocha to run (such as
utils.js) will be safely discarded.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub
<https://github.com/kaelig/sass-project-boilerplate/pull/1/files/4c6935f6f309b5617eadc1d7c7f49d23f64a9ede#r63369657>